<launch>
  <include file="$(find litterbox)/launch/experiment_deps.launch" />

  <node pkg="litterbox" type="pixel_occupied" name="pixel_occupied" respawn="false" output="screen" cwd="ROS_HOME">
    <param name="reference_x" value="2" />
    <param name="reference_y" value="0.075"/>
    <param name="unoccupied_mode" value="false"/>
    <param name="model_name" value="box1"/>
    <param name="reference_frame" value="/odom_combined"/>
    <param name="min_height" value="0.2"/>
    <param name="input_type" value="voxel"/>
    <remap from="collision_map_in" to="octomap_collision_map" />
    <!-- <remap from="collision_map_in" to="collision_map_occ"/> -->
    <remap from="voxel_grid_in" to="/move_base_node/local_costmap/voxel_grid"/>
    
  </node>

  <node pkg="costmap_2d" type="costmap_2d_markers" name="voxel_visualizer"> <remap from="voxel_grid" to="/move_base_node/local_costmap/voxel_grid"/> </node>
</launch>

